ambari-common/src/main/python/ambari_commons/credential_store_helper.py

```diff
@@ -50,7 +50,7 @@ def get_password_from_credential_store(alias, provider_path, cs_lib_path, java_h
     # Execute a get command on the CredentialUtil CLI to get the password for the specified alias
     java_bin = '{java_home}/bin/java'.format(java_home=java_home)
     cmd = (java_bin, '-cp', cs_lib_path, credential_util_cmd, 'get', alias, '-provider', provider_path)
-    cmd_result, std_out_msg  = checked_call(cmd)
+    cmd_result, std_out_msg  = checked_call(cmd, quite=True)
```

Comment:
quite ? Not quiet?

Reply:
Yup... I am one step ahead of you...  #1719